lib/better_errors/error_page_style.rb

```diff
@@ -1,9 +1,9 @@
-require "sassc"
-
 module BetterErrors
   # @private
   module ErrorPageStyle
     def self.compiled_css(for_deployment = false)
+      require "sassc" unless defined? SassC
```

Comment:
The condition here isn't strictly necessary, since multiple calls to `require` will short-circuit on second-and-subsequent invocations.

```suggestion
      require "sassc"
```

Reply:
Good point — I was being overly cautious here given my inability to get passing tests locally. I've updated this to remove the conditional.